CMSgov/easi-app
# [EASI-3052] - IT Gov admin action view

# EASI-3052

[Figma](https://jiraent.cms.gov/browse/EASI-3052)

## Changes and Description

- Created new view for admin action selection (src/views/GovernanceReviewTeam/Actions)
  - Moved V1 actions to different folder (src/views/GovernanceReviewTeam/ActionsV1)
- Added conditional route based on feature flag in src/views/GovernanceReviewTeam/RequestOverview.tsx
- Created components and routes for action forms
- Unit tests
- Updated test mock data



## How to test this change

- Log in as GRT admin
- Go to /governance-review-team/:systemIntakeId/actions
- Make sure action selection works as expected and goes to correct route when submitted



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
